free-3.md

```diff
@@ -57,12 +142,35 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this is a 50-50 language/system project. Rendering images is the most 
+"systems" aspect. And if I make a user interface, I'll spend a lot of time 
+working on that (I don't have much front-end experience). But I will have 
+to think a lot about the underlying language design. I have to consider what is 
+"intuitive" to humans (unions vs. OR, intersections vs. AND) and how to make it 
+easy to create intersections.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Venn diagrams should be simple enough to do in a half-semester. I'm worried 
+that this project is on the boring side, but maybe that's because I'm not a 
+target user. Rather, I think elementary teachers and business people are more 
+likely to use this project.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This project can make people's lives easier because they don't have to struggle 
+with other computer programs (or messily sketch by hand). It can add fluency to 
+otherwise unintuitive apps that focus on the exactly formatting of both the 
+circles and the text. My DSL would allow auto-formatting of the circle, 
+eliminating one user-controlled variable.
+
+On the other hand, I'll only be the 1000th person to try making venn diagrams 
+on a computer. Another drawback is that without a user interface, it'd be hard 
+for non-programmers to use my DSL. They won't learn programming languages just 
```

Comment:
I don't think this really is a drawback. The language snippet above is very very simple and I think it is complex enough to be a worry for people to learn